src/surface-components/Card/Card.module.css

```diff
@@ -0,0 +1,92 @@
+/* Card.module.css */
+
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Inter";
+}
+
+.cardWrapper {
+  width: 270px;
+  border: 1px solid black;
+}
+
+.cardWrapper:hover {
+  box-shadow: 4px 4px 0 #000000;
+}
+
+.imageWrapper {
+  overflow: hidden;
+}
+
+img {
+  position: center;
+  width: 270px;
+  height: 225px;
+  object-fit: cover;
+  object-position: center;
+}
+
+.fullWidth {
```

Comment:
In this component, you need to have both "fullWidth" (100%) and "fixedWidth". You should not assign "fullWidth" to  370px. It needs to be a reusable component and should always have a width of 100%, regardless of the content inside the card.